data/helpers.d/utils

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
-YNH_APP_BASEDIR=$(realpath $([[ "$(basename $0)" =~ ^backup|restore$ ]] && echo '../settings' || [[ -n "${YNH_ACTION:-}" ]] && echo '.' || echo '..' ))
+if [ -z "${YNH_APP_BASEDIR:-}" ];
+then
+    echo -e "YNH_APP_BASEDIR is not defined, many helpers use it, please define it!" >&2
+    YNH_APP_BASEDIR=$(realpath ..)
+fi
```

Comment:
After checking that we have put YNH_APP_BASEDIR everywhere we need to, I think we should change this part to :

```suggestion
YNH_APP_BASEDIR=${YNH_APP_BASEDIR:-$(realpath ..)}
```